src/gas/fetchBlockFeeHistory.ts

```diff
@@ -138,7 +136,7 @@ export default async function fetchBlockFeeHistory<Percentile extends number>({
   percentiles: givenPercentiles = [],
   includeNextBlock = false,
 }: {
-  ethQuery: EthQuery;
+  ethQuery: EthQueryish;
```

Comment:
An "EthQueryish" is an object that looks like an EthQuery without the full gamut of methods that EthQuery supports. Really all it needs is a `sendAsync` method. It's the type of the argument that `query` now takes.